frontend/src/api/mockAPI/positions.js

```diff
@@ -59,6 +59,9 @@ export class Position extends MockAPIController {
         });
     }
     getInstructors(position) {
+        if (!position.instructor_ids) {
+            return [];
+        }
         return position.instructor_ids.map(id =>
             new Instructor(this.data).find({ id })
         );
```

Comment:
`return !position.instructor_ids ? [] : position.instructor_ids.map(id => new Instructor(this.data).find({ id }))`

I write lots of one liners but I don't know if that's the preferred syntax for this project, so it's just an alternative.

Reply:
I actually don't have any preferences regarding the ternary operator(I go with everyone else's preferences lol). But for the same level of performance, I usually write something more straightforward and more readable (mostly for me).